pkgs/development/libraries/polkit/default.nix

```diff
@@ -110,18 +96,22 @@ stdenv.mkDerivation rec {
   inherit doCheck;
   checkInputs = [ dbus ];
   checkPhase = ''
+    runHook preCheck
+
     # unfortunately this test needs python-dbusmock, but python-dbusmock needs polkit,
     # leading to a circular dependency
     substituteInPlace test/Makefile --replace polkitbackend ""
 
     # tests need access to the system bus
     dbus-run-session --config-file=${./system_bus.conf} -- sh -c 'DBUS_SYSTEM_BUS_ADDRESS=$DBUS_SESSION_BUS_ADDRESS make check'
+
+    runHook postCheck
   '';
 
   meta = with lib; {
     homepage = "http://www.freedesktop.org/wiki/Software/polkit";
     description = "A toolkit for defining and handling the policy that allows unprivileged processes to speak to privileged processes";
-    license = licenses.gpl2;
+    license = licenses.lgpl2Only;
```

Comment:
According to https://gitlab.freedesktop.org/polkit/polkit/-/blob/025f9a6ba374c77c67f104ed5a4c04a057cc8084/meson.build#L4, it is lgpl2Plus.

Reply:
I'll update it to `lgpl2Plus`.

>The FSF uses the approach of asking people to release programs under “GNU GPL version N or any later version.” This >licensing is compatible with version N, and also with N+1 (because it offers version N+1 as an option). When you combine >code under “GPL 3 or later” with code under “GPL 2 or later”, the license of the combination is their intersection, which is >“GPL 3 or later”

https://www.gnu.org/licenses/license-compatibility.html

I managed to miss reading this clause in the `COPYING` file of the project.

https://gitlab.freedesktop.org/polkit/polkit/-/blob/025f9a6ba374c77c67f104ed5a4c04a057cc8084/COPYING#L392-403